src/styles/sass/base/_utilities.scss

```
@@ -17,3 +18,15 @@
         animation: popUp 0s;
     }
 }
+
+.scroll {
+    overflow: visible;
```

```suggestion
    overflow: scroll;
```

Should we prefer `visible` or `scroll`?